sdf/1.11/collision.sdf

```diff
@@ -0,0 +1,26 @@
+<!-- Collision -->
+<element name="collision" required="*">
+  <description>The collision properties of a link. Note that this can be different from the visual properties of a link, for example, simpler collision models are often used to reduce computation time.</description>
+
+  <attribute name="name" type="string" default="__default__" required="1">
+    <description>Unique name for the collision element within the scope of the parent link.</description>
+  </attribute>
+
+  <element name="laser_retro" type="double" default="0" required="0">
+    <description>intensity value returned by laser sensor.</description>
+  </element>
+
+  <element name="max_contacts" type="int" default="10" required="0">
+    <description>Maximum number of contacts allowed between two entities. This value overrides the max_contacts element defined in physics.</description>
+  </element>
+
+  <element name="material_density" type="double" default="1.0" required="0">
+    <description>Density of the collision geometry. Default is the density of water 1.0 kg/m^3</description>
```

Comment:
The density of water should be 1000 kg/m^3.

Reply:
Corrected the value in 62baec380b989918babb20fb235a414b81066f18